src/components/donation/Steps/Submit/Crypto/Crypto.tsx

```diff
@@ -1,128 +1,87 @@
-import { Info, LoadingStatus } from "components/Status";
+import { WalletProvider } from "@terra-money/wallet-provider";
```

Comment:
This component is previously `<Container/>` renamed to `Crypto` as it contains content which doesn't need `wallet`